tests/foreman/api/test_provisioning.py

```diff
@@ -117,7 +120,7 @@ def test_rhel_pxe_provisioning(
 
     # Run a command on the host using REX to verify that Satellite's SSH key is present on the host
     template_id = (
-        sat.api.JobTemplate().search(query={'search': 'name="Run Command - SSH Default"'})[0].id
+        sat.api.JobTemplate().search(query={'search': 'name="Run Command - Script Default"'})[0].id
```

Comment:
As this change is introduced in 6.12, it would not affect 6.11, can you revert this?

Reply:
Right, forgot about this change, Will update this. Thanks @Gauravtalreja1